pkg/api/testing/fuzzer.go

```diff
@@ -115,22 +115,24 @@ func FuzzerFor(t *testing.T, version string, src rand.Source) *fuzz.Fuzzer {
 		},
 		func(j *api.List, c fuzz.Continue) {
 			c.FuzzNoCustom(j) // fuzz self without calling this function again
-			if j.Items == nil {
-				j.Items = []runtime.Object{}
-			}
+			// TODO: uncomment when round trip starts from a versioned object
```

Comment:
Fixed

----- Original Message -----
> > @@ -115,22 +115,24 @@ func FuzzerFor(t *testing.T, version string, src
> > rand.Source) *fuzz.Fuzzer {
> >  		},
> >  		func(j *api.List, c fuzz.Continue) {
> >  			c.FuzzNoCustom(j) // fuzz self without calling this function again
> > -			if j.Items == nil {
> > -				j.Items = []runtime.Object{}
> > -			}
> > +			// TODO: uncomment when round trip starts from a versioned object
> 
> nit: prefer ```if false {``` to prevent bitrot.
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/GoogleCloudPlatform/kubernetes/pull/7490/files#r29353705